sentences/nl/_common.yaml

```diff
@@ -9,6 +9,7 @@ responses:
     handle_error: "Er is iets misgegaan"
 
 lists:
+  # brightness
```

Comment:
Stating the obvious 😅

Reply:
Yeah, the list (well actually a mapping) of lists became so long I wanted to bring some structure in it.
But I guess I overdid it :)